src/replica/Configuration.cc

```diff
@@ -762,23 +763,22 @@ DatabaseInfo& Configuration::_publishDatabase(util::Lock const& lock, string con
         }
         // Then publish the database.
         if (_connectionPtr != nullptr) {
-            _connectionPtr->executeInOwnTransaction([&](decltype(_connectionPtr) conn) {
-                conn->executeSimpleUpdateQuery(
-                        "config_database", conn->sqlEqual("database", databaseInfo.name),
-                        make_pair("is_published", 1), make_pair("publish_time", publishTime));
-            });
+            string const query = _g.update("config_database", make_pair("is_published", 1),
+                                           make_pair("publish_time", publishTime)) +
```

Comment:
`make_pair` feels like a noise. Could you write:

    _g.update("config_database", {"is_published", 1}, {"publish_time", publishTime})

Reply:
Well, this is the hardest request you've made on the PR. A problem is that, in C++, `std::make_pair` doesn't have the list initializer that would allow that. I already tried th ever same idea earlier (before making this PR). According to https://stackoverflow.com/questions/5290430/stdmake-pair-vs-c11-uniform-initializer
```
Using an initializer list constructor doesn't work for a pair, because a pair is heterogeneously typed,
while an initializer list constructor uses an initializer_list<T>, which is only used to retrieve
a homogeneously typed initializer list.
```
In my case, the first value is always a string (the name of a column), and the second value could be anything that would be serialized for insertion into the column. In theory, I could come up with a special class that would allow the list initializer. I will investigate this idea.